overview/features.rst

```diff
@@ -0,0 +1,103 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team Edition and Enterprise Edition.
+
+Team Edition Features
+----------------------------------------------
+
+1. **1-1 and group messaging**
+
+.. image:: ../images/messaging.png
+
+2. **File sharing**
+
+.. image:: ../images/file_sharing.png
+
+3. **Custom Emojis**
+
+.. image:: ../images/custom_emojis.png
+
+4. **Webhooks**
+
+.. image:: ../images/webhooks.png
+
+5. **Slash commands**
+
+.. image:: ../images/slash_commands.png
+
+6. **Plug-in framework**
+
+.. image:: ../images/plugin_framework.png
+
+7. **Mobile applications**
+
+.. image:: ../images/mobile_apps.png
+
+8. **Desktop applications**
+
+.. image:: ../images/desktop_apps.png
+
+9. **Multi-team support**
```

Comment:
May want to consider screenshot size here as well
![screenshot 2017-12-15 17 45 15](https://user-images.githubusercontent.com/9354841/34063585-f4cd6cae-e1c0-11e7-8795-bbf5c5dfe9c0.png)